op-scim-bridge/templates/secret.yaml

```diff
@@ -1,7 +1,9 @@
-{{- if .Values.scim.credentialsSecret }}
-{{- if or .Values.scim.credentialsSecret.value_json .Values.scim.credentialsSecret.value_base64 }}
-{{- if and .Values.scim.credentialsSecret.value_json .Values.scim.credentialsSecret.value_base64 }}
-  {{- fail "Only one of scim.credentialsSecret.value_json and scim.credentialsSecret.value_base64 can be specified" }}
+{{- if .Values.scim.credentialsSecret.enabled }}
+{{- if and .Values.scim.credentialsSecret.enabled .Values.scim.credentialsVolume.enabled }}
+  {{- fail "Either credentialsSecret or credentialsVolume may be specified, not both"}}
+{{- end }}
```

Comment:
Fantastic. Thanks for this change. 🎉  I believe our exiting check may not have worked as desired.